Userland/Libraries/LibWeb/DOM/DocumentLoading.cpp

```diff
@@ -335,6 +335,24 @@ static WebIDL::ExceptionOr<JS::NonnullGCPtr<DOM::Document>> load_media_document(
         return {};
     };
 
+    auto style_element = TRY(DOM::create_element(document, HTML::TagNames::style, Namespace::HTML));
+    style_element->set_text_content(R"~~~(
+        :root {
+            background-color: #222;
+        }
+        img, video, audio {
+            position: absolute;
+            inset: 0;
+            max-width: 100vw;
+            max-height: 100vh;
+            margin: auto;
+        }
+        img {
+            background-color: #fff;
+        }
+    )~~~"_string);
+    TRY(document->head()->append_child(style_element));
```

Comment:
These both seem like `MUST` situations, not `TRY`, right?
Under no circumstances should these operations fail.

Reply:
All the code around this change use `TRY`, you are right that this should be unfailable